.github/workflows/hosted-notebook-samples-check.yml

```diff
@@ -4,17 +4,75 @@ on:
   pull_request:
     branches: 
       - feature/samples-gallery
```

Comment:
We might need to change this

Reply:
I'm planning to add some filtering to the devops pipeline, and then I plan to address this in future PR!